### ISSUE TITLE
Reconsider donor model

### ISSUE THREAD

**Issue by MilesBHuff** [SettleScape/server#60]
Currently, the donation model is that donors are a small group of elites who get a handful of perks, and a say in the project's direction.  The goal is to always be revenue neutral.

However, when we go public, people are going to want to become donors in such numbers that turning a profit will be inevitable, because we can't reasonably lower the donor price below a dollar.  As such, we need to answer some questions.

1. Should there be a maximum number of donors?
2. What should the price be?
3. What should we do with the profits?

With 1, I'm thinking no?  I mean, if someone is a core member of the community, we'd generally always want them to be able to donate.
Maybe a good middle-ground is to require a certain minimum rank to be able to donate?  I don't know how to actually hard-restrict that, though, and giving refunds sucks.
Maybe just make it so you have to have a certain rank, in addition to donating, to be able to vote on the project's future.  That would prevent someone from sockpuppetting the server, since it takes effort to rank up.  It would rightly upset low-rank donors, though.
The lowest we could do is Traveller;  but actually I think Settler might be better.  Not only is it the server's namesake, but it is also the rank at which we trust you with lava.
Maybe project governance should not be a perk of donating;  but rather that donating is a requirement for it.

For 2, we shouldn't set it too low, since donating does provide tangible benefits.  If it's only a dollar, then someone can easily sockpuppet server governance (This can be mitigated by restricting voting to certain rank;  see above.).  Maybe $3 is reasonable?

For 3, the ethical thing is to invest the profits first into the server, then to plugins we depend on (so, we would donate to, for example, Towny), and then to project contributors (probably in proportion to number of commits?).  In this way, not only would we be able to give back to the open source projects that made our server a success, but we'd also be able to incentivize future development.
A requirement to receive dev money would be to have a GitHub Sponsors account.
Wiki contributors should be counted for this purpose.
We should create a budget and percentage allocations for different purposes;  this will be discussed later.

**Comment by MilesBHuff** [comment]
@slankford @Middie321 @AutoSec7

**Comment by MilesBHuff** [comment]
tl;dr:
* Change the project governance to be a Senate composed of all Settler-rank donors.
* Set donation price to a flat $3 or so.
* Profits get redonated to projects we depend on, and split among project contributors (to encourage people to help contribute) (including wiki authors)

**Comment by MilesBHuff** [comment]
I'm going with what I discussed above.
